packages/client.api/src/Definition.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type {
+  ActionDefinition,
+  InterfaceDefinition,
+  ObjectTypeDefinition,
+  QueryDefinition,
+} from "@osdk/api";
+
+export type Definition =
```

Comment:
I need this type in order to type the parameters for client. I can't find a type like this already in the api. I'm not sure if this is the correct name or if I should name it something like MetadataDefinition.

Reply:
Hmm. I'm on the fence about having a type like this for minimal use

Reply:
Ok, I can just inline the type in the 2 places that need it.